plasmapy/mathematics/mathematics.py

```diff
@@ -91,7 +91,8 @@ def plasma_dispersion_func(zeta: Union[complex, int, float, np.ndarray, u.Quanti
         raise ValueError("The argument to plasma_dispersion_function is "
                          "not finite.")
 
-    Z = 1j * np.sqrt(np.pi) * Faddeeva_function(zeta)
+    # wofz is the so-called Faddeeva function
+    Z = 1j * np.sqrt(np.pi) * scipy.special.wofz(zeta)
```

Comment:
I'm a little hesitant about this change.  The root of the problem is that `wofz` is a bad name for a function since it gives very little indication of what it does except to people who are already familiar with the notation.  It took me a while to even realize that it means "w of z".  One of the best practices that I've found in a few places is to write the code so that it says how it work rather than relying on comments.  Maybe a compromise would be to change `Faddeeva_function` in the original to `scipy_fadeeva_function` so that it shows that it is from SciPy and also provides the name?

Reply:
I just realized http://docs.plasmapy.org/en/master/mathematics/index.html doesn't get `Fadeeva` in the namespace because we're importing from `mathematics/__init__.py`, not `mathematics.py`, so this change is completely useless. Best to skip it.